internal/pkg/plugin/githubactions/golang/templates.go

```diff
@@ -1,11 +1,21 @@
 package golang
 
-var PrBuilder = `
-name: PR Builder
+var PrMasterBuilder = `
```

Comment:
Change variable name and job name to a better one.

Reply:
done, update it to "PipelibeBuilder"